65black/StackOverflow-lite
# #160818244 Updates Search Functionality To TS_VECTOR

## What does this PR do?
Currently, searching for a question only matches the question content and not the question title.
The search functionality is also being implemented with the primitive ILIKE. 

This chore aims to fix these problems by switching to using ts_vector to handle search functionality.

### Description of Task to be completed?
 - Edits DB to accommodate TS_VECTOR columns
 - Edits the search controller to search both title and content fields

### How should this be manually tested?
After cloning the repo, cd into it and run `npm run start-dev`. Follow these steps:

 - Open Postman and make any search.
 - You should get better and improved results.

### Screenshots
N/A

### What are the relevant pivotal tracker stories?
#160818244
